apps/roboshield/payload.config.ts

```diff
@@ -42,7 +43,12 @@ const smtpHost = process.env.SMTP_HOST || "smtp.sendgrid.net";
 
 export default buildConfig({
   serverURL: appURL,
-  editor: slateEditor({}),
+  editor: lexicalEditor({
+    features: ({ defaultFeatures }) => [
+      ...defaultFeatures,
+      SlateToLexicalFeature({ disableHooks: true }),
```

Comment:
Why do we need this?

Reply:
We need to add `SlateToLexicalFeature` to enable automatic preview of slate content in the admin panel. As per the [migration instructions](https://payloadcms.com/docs/rich-text/migration#migration-via-migration-script-recommended) we are supposed to disable hooks when we run the migration

Reply:
Thanks @m453h . Is there a way of us to know whether we're supposed to remove that code after the migration or not?

Reply:
... cool, then lets remove it @m453h

Reply:
I'll keep a TODO note and we'll remove after we migrate PROD db